USAGE.md

```diff
@@ -100,6 +101,11 @@ end
 Packwerk::Parsers::Factory.instance.erb_parser_class = CustomParser
 ```
 
+## Using the experimental cache
+Packwerk ships with an experimental cache to help speed up file parsing. You can run packwerk with `EXPERIMENTAL_PACKWERK_CACHE=1 bin/packwerk ...` to turn this on (per user) or by setting `experimental_cache: true` in `packwerk.yml` to turn it on for all users.
+
+This will write to `tmp/cache/packwerk`. It is not recommended to use the cache in your test suite, as this cache is experimental and it is best to ensure CI is running with no cache.
```

Comment:
Good point. We originally had it listed as not recommended and experimental because there were some known cases where the cache would become invalid but not properly busted. Now that we've addressed that and are considering issues with the cache as unexpected bugs, we can remove this disclaimer.